components/WelcomeScreen.js

```diff
@@ -0,0 +1,28 @@
+import { Button, Jumbotron } from 'react-bootstrap'
+
+const WelcomeScreen = ({ handleSignup }) => (
+  <>
+    <Jumbotron>
+      <h1>Welcome to the {process.env.SITE_TITLE}!</h1>
```

Comment:
Should probably just make all of the h1 contents be the title environment variable.

Reply:
Not sure what you mean here. Are you saying remove the `Welcome to the...`? Maybe we should just add this to the internal testing pile.

Reply:
Yeah, I meant put all of that into the SITE_TITLE string.

Reply:
Ah, I don't think we should do that because `SITE_TITLE` is also used in the `<title>` tag in `LayoutWithAuth0.js`.